lib/gui-utils/src/GUIIcons.h

```diff
@@ -62,4 +62,85 @@ static const unsigned char PROGMEM ic_sensor_live[] = {
     B10101010,
     B01010100,
     B00000000,
-};
\ No newline at end of file
+};
+
+#ifdef Emoticons
```

Comment:
The directives of preprocessor should be in upper case

```suggestion
#ifdef EMOTICONS
```

Reply:
Directives of preprocessor in upper case:
https://github.com/danielbernalb/esp32-hpma115s0/commit/d78dabae9444e2f0b8892aacb3804c0599a01315